### DISCUSSION TITLE
Duda con entrega de tag entrega final

### DISCUSSION THREAD

**Discussion by tobiascastaneda2000** [dds-jv/foro#399] [Consultas de herramientas] [answered]
Buenas Tardes!

Quería preguntar una duda antes de dejas la entrega de tp individual macowins: 
Realice el tag con éxito y ya esta subido. Sin embargo, me dijeron algunos compañeros que del lado derecho tendría que tener una tilde verde con un verify, el cual no me aparece en mi caso.
![tag](https://user-images.githubusercontent.com/62578744/236637647-575ea811-c889-467f-8aab-05316717a276.png)

Debe tener la tilde verde para estar entregado o ya estaría correcta la entrega? 

Disculpe las molestias, espero su respuesta.

**Comment by flbulgarelli** [comment, marked as answer]
¡Hola!

El tilde verde debe aparecer en el commit que se ve desde la vista principal del repositorio, no desde el listado de tags. Entre a tu repositorio y constatá que el último commit, o en todo caso, al que apunta el tag, tenga un tilde verde y no una cruz.
